### PR TITLE
Feature/add uk postcode validator

### PR DESCRIPTION
**Pull request by rickclare**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -3,7 +3,6 @@
 require File.expand_path("../dummy/config/environment.rb",  __FILE__)
 
 require 'rspec/rails'
-require 'rspec/autorun'
```
Comment (Cruikshanks) [inline spec/spec_helper.rb]: Don't know enough about rspec to know what this was doing, or why we are taking it out? Can you elaborate?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -28,7 +27,7 @@
 
   config.include FactoryGirl::Syntax::Methods
 
-  ActiveRecord::Migration.maintain_test_schema!
```
Comment (Cruikshanks) [inline spec/spec_helper.rb]: Similar to above don't know enough about rspec to know what this was doing, or why we are commenting it out? Can you elaborate?

File: spec/spec_helper.rb (hunk 1 above)
Comment (rickclare) [inline spec/spec_helper.rb]: When running `rspec` I was getting the following warning:

> Requiring `rspec/autorun` when running RSpec via the `rspec` command is deprecated.

Therefore I removed it.

File: spec/spec_helper.rb (hunk 2 above)
Comment (rickclare) [inline spec/spec_helper.rb]: I was having a problem executing the specs, when this line was it.  However, I now think this wasn't the cause of my problems, so I will add back in.

**Pull request merged**